mkdocs.yml

```diff
@@ -114,22 +128,31 @@ plugins:
   - social
   - mknotebooks
   - tags:
-      tags_file: tutorials/index.md
+      tags_file: guides/tutorials/index.md
 
 
 nav:
-  - Argilla: index.md
+  - Home: index.md
   - Getting started:
     - Quickstart: getting_started/quickstart.md
     - Installation: getting_started/installation.md
     # - FAQ: getting_started/faq.md
-  - How-to guides:
-    - how_to_guides/index.md
-    - Manage users and credentials: how_to_guides/user.md
-    - Manage workspaces: how_to_guides/workspace.md
-    - Manage and create datasets: how_to_guides/dataset.md
-    - Add, update, and delete records: how_to_guides/record.md
-    - Query, filter, and export records: how_to_guides/query_export.md
+  - Guides:
+    - guides/index.md
+    - How-to guides:
+      - guides/how_to_guides/index.md
+      - Manage users and credentials: guides/how_to_guides/user.md
+      - Manage workspaces: guides/how_to_guides/workspace.md
+      - Manage and create datasets: guides/how_to_guides/dataset.md
+      - Add, update, and delete records: guides/how_to_guides/record.md
+      - Query, filter, and export records: guides/how_to_guides/query_export.md
+    - Tutorials:
+      - guides/tutorials/index.md
+      - Bulk labelling through vector search and fine-tuning a few-shot text classifier: guides/tutorials/few-shot-textcat-with-setfit.ipynb
```

Comment:
I would not expose this tutorial until it's finished. Especially since it requires a menu change.

Reply:
This is the tutorial I want to wrap up with the new SDK to get familiar with it and only merge after having done this.

Reply:
That's cool. Maybe hold off on integrating it in/ changing the menu then.